plugin/mouse/mouse.py

```diff
@@ -239,6 +247,14 @@ def hiss_scroll_down():
         global hiss_scroll_up
         hiss_scroll_up = False
 
+    def pop_twice_to_wake():
+        """Use pop sound to wake from sleep"""
+        global time_last_pop
+        delta = time.time() - time_last_pop
+        if delta >= 0.1 and delta <= 0.3:
```

Comment:
Is the minimum here to prevent false positives? Guessing you just grabbed this from @AndreasArvidsson's impl?

Reply:
Correct this is from my implementation.

Reply:
Cool. What's that 0.1 min for?

Reply:
As you said: to prevent false positives

Reply:
yes - copy from Andreas - so nothing to do here I guess - will mark as resolved